prov/efa/src/rxr/rxr_init.c

```diff
@@ -239,7 +239,7 @@ void rxr_info_to_core_mr_modes(uint32_t version,
 					hints->domain_attr->mr_mode & OFI_MR_BASIC_MAP;
 			core_info->addr_format = hints->addr_format;
 		}
-#if HAVE_LIBCUDA
+#if defined(HAVE_LIBCUDA) || defined(HAVE_NEURON)
```

Comment:
HAVE_NEURON and HAVE_LIBCUDA is always defined, right? just the value is 0 or 1.  should it be
```
#if HAVE_LIBCUDA || HAVE_NEURON
```

?